packages/libs/react-components/src/components/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import { Button, SystemIcons } from './../../';
+import { colorVariant, simpleVariant } from './styles';
+import { INotificationProps, Notification } from '.';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  {
+    selectIcon: keyof typeof SystemIcons;
+    text: string;
+  } & INotificationProps
+> = {
+  title: 'Layout/Notification',
+  argTypes: {
+    selectIcon: {
+      options: Object.keys(SystemIcons) as (keyof typeof SystemIcons)[],
+      control: {
+        type: 'select',
+      },
+    },
+    title: {
+      control: {
+        type: 'text',
+      },
+    },
+    description: {
+      control: {
+        type: 'text',
+      },
+    },
+    simple: {
+      options: Object.keys(simpleVariant) as (keyof typeof simpleVariant)[],
+      control: {
```

Comment:
simple is a boolean. can we make the control also a boolean like displayCloseButton option?